analyze-project/action.yml

```
@@ -0,0 +1,94 @@
+name: Analyze Python Project
```

The other action names are in sentence case and match the action subdirectory. We don't need to Smurf-name everything with "Python".
```suggestion
name: Analyze project
```